swagger.yml

```diff
@@ -1014,6 +1014,34 @@ paths:
             $ref: '#/definitions/plugin'
         default:
           $ref: '#/responses/error'
+  /sites/{site_id}/plugin_runs/latest:
+    get:
+      operationId: getLatestPluginRuns
+      tags: [x-internal]
+      description: 'This is an internal-only endpoint.'
+      parameters:
+        - name: site_id
+          in: path
+          type: string
+          required: true
+        - name: packages
+          in: query
+          type: array
+          items:
+            type: string
+          required: true
+        - name: state
```

Comment:
Is the default behavior "do not filter by `plugin_run.state`"?

Reply:
yep!